src/patches.rs

```diff
@@ -244,6 +244,52 @@ fn patch_artifact_totem_scan_strg(res: &mut structs::Resource, text: &str)
     Ok(())
 }
 
+fn build_lore_scan_strings<R>(
```

Comment:
Several unused variables on this function. If you're planning on actually using the variables in the future, add an underscore to the front of the variable to suppress the Clippy errors.